override_values.tf

```diff
@@ -608,3 +608,29 @@ resources:
   EOT
   filename = "${path.module}/override_vales/kube_state_metrics.yaml"
 }
+
+
+resource "local_file" "keda_helm_config" {
+  count    = var.keda && (var.keda_helm_config == null) ? 1 : 0
+  content  = <<EOT
+global:  
```

Comment:
This `global` variable is not present in default keda values,yaml, please fix it.

Reply:
done